pyexasol/http_transport.py

```diff
@@ -17,21 +22,84 @@ def __init__(self, connection, compression):
         self.connection = connection
         self.compression = compression
 
-        self.params = {}
+        self.params: dict = {}
         self.http_thread = None
-        self.exa_address_list = []
+        self.exa_address_list: list = []
         self.exc = None
 
         super().__init__()
 
-    def set_http_thread(self, http_thread):
+    def set_http_thread(self, http_thread) -> None:
         self.http_thread = http_thread
         self.exa_address_list = [http_thread.exa_address]
 
-    def set_exa_address_list(self, exa_address_list):
+    def set_exa_address_list(self, exa_address_list) -> None:
         self.exa_address_list = exa_address_list
 
-    def run(self):
+    @property
+    def certificate(self) -> Optional[str]:
+        """
+        Pyexasol uses a self-signed certificate for import & export.
```

Comment:
Is there a 7.x version from which this is cut off?
The PUBLIC key is only support from 8.32 & not the 7.x, right?